docs/architecture/decisions/0002-generating-unique-ids.md

```diff
@@ -0,0 +1,41 @@
+# 1. Record architecture decisions
+
+Date: 2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+When users start an LPA either online or via the paper route, their data will be sent to and stored in this ID service data store. We need to create a unique ID to store alongside the users data, which case workers can then use to find LPA's by when actioning any future enquiries or objections.
+
+We must define the format and method of generating these IDs. This can change for private beta, but should preferably be something we don’t need to change until then.
+
+The IDs generated must:
+- Only use alphanumeric characters
+- Be unique
+- Have a clear prefix to differentiate them from existing IDs
+
+
+For private beta (but not necessarily now) they also must:
+- Not be calculable (e.g. not a sequence)
+- Use the safe alphabet identified by Use an LPA `346789QWERTYUPADFGHJKLXCVBNM`
```

Comment:
You should probably explain the why of these. i.e. avoid iteration based attacks and avoid typographically similar letters i,l,1

Reply:
Good point, i'll add some explanations in 👍🏼